Source/Modules/php.cxx

```diff
@@ -2007,6 +2013,10 @@ class PHP : public Language {
 	  if (value) {
 	    Printf(pragma_phpinfo, "%s\n", value);
 	  }
+	} else if (Strcmp(type, "version") == 0) {
+	  if (value) {
+	    Printf(pragma_version, "%s", value);
```

Comment:
You've not done what I suggested here - did you try it and find it didn't work?

If we do it your way, then ideally we don't leak memory if this pragma is used more than once.  Also, using `Copy()` would be a clearer way to make a copy.